2_process/src/munge_inst_timeseries.R

```diff
@@ -55,3 +55,36 @@ aggregate_data_to_hourly <- function(inst_data,output_tz){
 
 }
 
+aggregate_data_to_daily <- function(inst_data, daily_data, output_tz){
+  #' 
+  #' @description Function to aggregate instantaneous NWIS data collected at sub-hourly (e.g. 15/5/30 min) intervals to hourly min/mean/maxs
+  #'
+  #' @param inst_data a data frame containing the downloaded time series for NWIS instantaneous site. 
+  #' inst_data must include the following columns: c("Value_Inst","dateTime","site_no")
+  #' @param daily_data a data frame the downloaded daily time series of DO data. This is used so that if a site is already in 
+  #'  the daily sites, we won't do the aggregating here
+  #' @param output_tz character string to set display attribute of dateTime. Possible values to provide are "UTC",
+  #' "America/New_York","America/Chicago", "America/Denver","America/Los_Angeles", "America/Anchorage", 
+  #' as well as the following which do not use daylight savings time: "America/Honolulu", "America/Jamaica",
+  #' "America/Managua","America/Phoenix", and "America/Metlakatla"
+  #'
+  #' @value A data frame containing daily min, mean, and max values the parameter of interest
+  
+  
+  only_inst_data = setdiff(inst_data$site_no, daily_data$site_no)
+  
+  daily_values <- inst_data %>%
+    filter(site_no %in% only_inst_data) %>%
+    mutate(dateTime = with_tz(dateTime,tzone=output_tz)) %>%
```

Comment:
I'd suggest the following minor changes for code readability:

- throughout the repo, I've tried to indicate the function source when we're using packages outside of base and tidyverse. We could indicate the source of the time parsing functions with `lubridate::with_tz(...)` and `lubridate::date()`
- the two calls to `mutate` can be combined into one call, with the two new variables separated by a comma
- I'd suggest replacing `dateTime` with `dateTime_local` or something similar that more readily communicates that we're changing the time zone for the daily aggregation (rather than writing over the original time stamps)